packages/amplication-client/src/Plugins/InstalledPlugins.tsx

```diff
@@ -52,9 +53,19 @@ const InstalledPlugins: React.FC<Props> = ({ match }: Props) => {
     },
   });
 
+  const { data: resourceSettings } = useQuery<{
+    serviceSettings: models.ServiceSettings;
+  }>(GET_RESOURCE_SETTINGS, {
+    variables: {
+      id: resource,
+    },
+  });
+
   const userEntity = useMemo(() => {
     return entities?.entities?.find(
-      (entity) => entity.name.toLowerCase() === USER_ENTITY_NAME
+      (entity) =>
+        entity.name.toLowerCase() ===
+        resourceSettings.serviceSettings.authEntityName?.toLowerCase()
```

Comment:
don't you want to listen to: resourceSettings.serviceSettings.authEntityName inside your memo ?